### PR TITLE
Introductory Pages Review

### PR DESCRIPTION
**Pull request by marcdrury**
Reviewed `Introduction`, `Why Athena` and `Getting Started` pages.

## Additional Notes ##
- The Documentation Conventions section of the Introduction can be confusing because the navigation bar it mentions is hidden once you scroll down, which is required to get to the section.

- I don't understand the last two sentences of the third paragraph of the Error Handling section on the Getting Started page. What happens if the exception goes unhandled? It currently reads:
  *"If the exception goes unhandled, i.e. no listener sets an [ATH::Response](https://athenaframework.org/Framework/Response/#Athena::Framework::Response). By default, non [ATH::Response](https://athenaframework.org/Framework/Response/#Athena::Framework::Response) on the event, then the request is finished and the exception is re-raised."*

### CODE REVIEW

**Comment by Blacksmoke16** [comment]
> * The Documentation Conventions section of the Introduction can be confusing because the navigation bar it mentions is hidden once you scroll down, which is required to get to the section.

Possibly could handle this via https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#sticky-navigation-tabs. But would reduce the reading area by 3 lines :/. Maybe would be enough to just do like:

```diff
- The navigation bar towards the top of the screen includes a tab for each component representing that component's API documentation.
+ When scrolled to the top of the page, the navigation bar includes a tab for each component representing that component's API documentation.
```

> * I don't understand the last two sentences of the third paragraph of the Error Handling section on the Getting Started page. What happens if the exception goes unhandled? It currently reads:

I'd reword that to like: 

> If the exception goes unhandled, i.e. no listener sets an `ATH::Response` on the event, then the request is finished and the exception is re-raised.

**Comment by marcdrury** [comment]
Sounds good to me, I made an additional commit adding those changes.
